dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -196,13 +204,20 @@ export const ManyNewsletterSignUp = () => {
 	}, [toggleNewsletter, newslettersToSignUpFor]);
 
 	const sendRequest = async (reCaptchaToken: string): Promise<void> => {
-		reportTrackingEvent('ManyNewsletterSignUp', 'form-submit', {
-			newsletterIds: newslettersToSignUpFor,
+		const identityNames = newslettersToSignUpFor.map(
+			(newsletter) => newsletter.identityName,
+		);
+		const idNumbers = newslettersToSignUpFor.map(
+			(newsletter) => newsletter.idNumber,
+		);
```

Comment:
not super important, but we could avoid going over the list twice with a `reduce`, but that's not always easy to read!